packages/core/src/execution-engine/workflow-execute.ts

```diff
@@ -1429,16 +1429,13 @@ export class WorkflowExecute {
 						node: executionNode.name,
 						workflowId: workflow.id,
 					});
-					await hooks.runHook('nodeExecuteBefore', [executionNode.name, taskStartedData]);
```

Comment:
Then the log at 1428 should move as well or?

Reply:
I don't have a strong opinion about that. It says "processing" which is true, the node is being processed, but not yet executed. 🤷🏾

Reply:
done